site/static/examples/sample-validating-policy-binding.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: sample-validating-admission-policy-binding
+spec:
+  policyName: sample-validating-admission-policy
+  validationActions: [Deny]
```

Comment:
Does this policy match every namespace? If so, I suggest making the example match a specific namespace or a namespace label.

Reply:
The example has been updated to exclude only a specific namespace from validation.